src/Components/Pages/LoginForm/LoginForm.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { useState } from 'react'
+
+export default function LoginForm() {
+    const [username, setUsername] = useState("")
+
+    const handleChange = (event) => {
+      setUsername(event.target.value)
+    }
+  
+    const handleSubmit = (event) => {
+      event.preventDefault();
+      alert(`Bonjour ${username}`);
+      setUsername("")
+    }
+  return (
+    <div className="LoginForm">
+        <form action='submit'
+          onSubmit={handleSubmit}>
+        <h2>Bienvenue chez nous !</h2>
+        <br />
+        <h3>Connectez vous</h3>
+
+        <input type="text" 
+          id="name" 
+          placeholder='Entrez votre prenom'
```

Comment:
Idem il te manque les "..." après "Entrez votre prénom"
```suggestion
          placeholder='Entrez votre prenom...'
```

Reply:
Ok super,  merci beaucoup Tom ! Je fais les changements de suite !